src/NuGet.Clients/NuGet.VisualStudio.OnlineEnvironment.Client/SolutionExplorer/PackageManagerUICommandHandler.cs

```diff
@@ -248,8 +249,8 @@ private async Task ShowPackageManagerUI(string projectPath)
             return null;
         }
 
-        private async Task<System.Windows.Window> CreateNewWindowFrameAsync(string projectPath)
-        // private async Task<vsShellInterop.IVsWindowFrame> CreateNewWindowFrameAsync(string projectPath)
+        // private async Task<System.Windows.Window> CreateNewWindowFrameAsync(string projectPath)
```

Comment:
let's delete all the "window" or other workaround code.

Reply:
Will do. There's a ton of cleanup that needs to happen anyway.